galaxy_importer/main.py

```diff
@@ -50,6 +48,26 @@ def main(args=None):
     write_output_file(data)
 
 
+def setup_logger(cfg):
+    """Sets up logger with custom formatter."""
+    logger = logging.getLogger()
+    logger.setLevel(getattr(logging, cfg.log_level_main, 'INFO'))
+
+    ch = logging.StreamHandler(stream=sys.stdout)
+    ch.setFormatter(CustomFormatter())
+    logger.addHandler(ch)
```

Comment:
[this comment doesnt require any changes just figured I'd mention it]

In theory, could do this with the default Formatter (s) by providing two handlers on the default logger. One that handles only 'INFO' level by adding a filter that drops everything except records at 'INFO'. And the other handler would handle everything else by using a filter that drops 'INFO' records. (similar to the django 'RequireDebugFalse' filter https://docs.djangoproject.com/en/3.0/topics/logging/#django.utils.log.RequireDebugFalse).

That said, I'm not sure my idea is a good idea. The main advantage is it makes it possible to further tweak the log setup with logging dictConfig. But for this case my idea would be over complicated especially since the builtin logging dictConfig isn't being used.